setup.cfg

```diff
@@ -14,7 +14,3 @@ license_file = LICENSE.txt
 [bdist_wheel]
```

Comment:
do you mind removing this as well? there is no need for a universal wheel anymore, we do not support python 2 anylonger.